### PR TITLE
Mj/use xrpljs

### PR DESCRIPTION
**Pull request by mukulljangid**
## High Level Overview of Change
Replace `gRPC` with `WebSocket`.

### Context of Change
`gRPC` is deprecated. This task is accomplished by using `xrpl.js` instead of the old `xpring-js`.

### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Chore (project configuration, scripts, dotfiles, etc.)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation Updates
- [ ] Release

## Before / After
Functionalities work as expected.

## Test Plan
Passes CI tests. (Tests for the functions that were implemented in this app are deleted.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Prompt overrides help:
   inputCsv - The input CSV path. Should point to a file that contains the receiver's XRP details.
   outputCsv - The output CSV path. Will be generated after xrp-batch-payout is complete.
   network - The XRPL network. Either 'testnet' or 'mainnet'.
-  grpcUrl - The web gRPC endpoint for the rippleD node.
+  serverUrl - The web gRPC endpoint for the rippleD node.
```
Comment (theotherian) [inline README.md:37]: Comment is wrong; it's no longer a gRPC node.

File: src/apps/payout.ts [hunk 2]
```diff
@@ -57,27 +54,27 @@ export default async function payout(override?: unknown): Promise<void> {
...
-    const [xrpNetworkClient, balance] = await connectToLedger(
-      senderInput.grpcUrl,
-      senderInput.network,
-      classicAddress,
-    )
+    
     log.info(green(`Connected to XRPL ${senderInput.network}.`))
     log.info(
-      black(`  -> RippleD node web gRPC endpoint: ${senderInput.grpcUrl}`),
+      black(`  -> RippleD node web gRPC endpoint: ${senderInput.serverUrl}`),
```
Comment (theotherian) [inline src/apps/payout.ts:69]: This info needs to be updated too.

File: src/apps/payout.ts [hunk 3]
```diff
@@ -57,27 +54,27 @@ export default async function payout(override?: unknown): Promise<void> {
...
     )
-    log.info(black(`  -> ${classicAddress} balance: ${balance} XRP`))
+
+    const client = new Client(senderInput.serverUrl)
+    await client.connect()
```
Comment (theotherian) [inline src/apps/payout.ts:73]: Maybe a try/catch here in case it fails so we can write out a log message?

File: src/lib/config.ts [hunk 4]
```diff
@@ -3,9 +3,9 @@ import prompts from 'prompts'
 import { XrplNetwork } from 'xpring-js'
```
Comment (theotherian) [inline src/lib/config.ts:3]: We should be able to remove these imports now as well as the dependency.

**Review by sappenin — APPROVED** [review]
LGTM!

**Review by theotherian — CHANGES_REQUESTED** [review]
A few minor things to address but looks good otherwise. I'd like @florent-uzio to have a look too before merging.

File: README.md (hunk 1 above)
Comment (mukulljangid) [reply README.md:37]: I agree

File: src/apps/payout.ts (hunk 3 above)
Comment (mukulljangid) [reply src/apps/payout.ts:73]: This entire function is in a try-catch already and errors seem to be getting caught.

File: README.md [hunk 5]
```diff
@@ -44,7 +49,7 @@ Example prompt overrides object:
   "inputCsv": "input.csv",
   "outputCsv": "output.csv",
   "network": "mainnet",
-  "grpcUrl": "https://envoy.main.xrp.xpring.io",
+  "serverUrl": "https://s1.ripple.com",
```
Comment (florent-uzio) [inline README.md:52]: The WebSocket should be `wss://s1.ripple.com`.

**Review by florent-uzio — CHANGES_REQUESTED** [review]
Good job! I left a few comments, some things to change in my opinion.

**Review by florent-uzio — APPROVED** [review]
LGTM

**Review by theotherian — APPROVED** [review]

File: src/apps/payout.ts (hunk 3 above)
Comment (theotherian) [reply src/apps/payout.ts:73]: My mistake there; sorry!

**Pull request merged**